README.md

```diff
@@ -1,15 +1,15 @@
 # htmlq
-Like jq, but for HTML. Uses CSS selectors to extract bits of content from HTML files. Mozilla's MDN has a good <a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors">reference for CSS selector syntax</a>.
+Like [`jq`](https://stedolan.github.io/jq/), but for HTML. Uses [CSS selectors](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors) to extract bits of content from HTML files.
 
 ## Installation
 
-```
+```sh
```

Comment:
I'm curious, why it is `sh` here while `console` elsewhere?

Reply:
`console` is used for shell-sessions, where every line that doesn't begin with a prompt-like symbol gets highlighted as program output:

~~~console
# echo "Foo"
Foo
$ echo "Bar"
Bar
% echo "Baz"
Baz
λ echo "cargo install htmlq"
cargo install htmlq
~~~

In contrast, `sh` just highlights code like an ordinary shell-script ([aliases](https://github.com/github/linguist/blob/bc56fd06e3b9669adfd4ae301ac2d14b68d71517/lib/linguist/languages.yml#L5561-L5565) include `bash` and `shell`).

Reply:
Thanks for explanation!